elastic/integrations
# move Windows logs to system

## What does this PR do?

This PR is meant to provide a functional OOTB logging experience for Integrations on windows. More precisely, we're moving most of the logs from the Windows integration (save for the powershell logs) over to the system integration. Right now, there's no OOTB logging on windows, as users would have to manually add the Windows integration. I'm not a windows expert, and I'm still working on testing this with actual data, but right now the system integration installs everything and isn't broken pre-ingest.

This is also sort of a draft, as I want other people's input as to what I've migrated and if there's certain log data streams that we either want to migrate back to windows, or add to system.

## Checklist

- [x] I've updated the docs

## How to test this PR locally

- I tested this with https://github.com/elastic/elastic-package
- Once you spin everything up, and have an agent running on windows, we're looking to make sure:
  - All three windows log sources (application, system, security) are collected on windows
  -  The agent isn't trying to start the log sources on non-windows hosts. Look for a "module X not found" error in the agent logs.

## Related issues

- https://github.com/elastic/integrations/issues/399

## Screenshot
<img width="1040" alt="Screen Shot 2020-12-10 at 11 43 26 AM" src="https://user-images.githubusercontent.com/8418476/101821569-fbffe600-3adc-11eb-988f-f9c3d2252dfe.png">

<img width="1294" alt="Screen Shot 2020-12-10 at 11 53 07 AM" src="https://user-images.githubusercontent.com/8418476/101822529-4b92e180-3ade-11eb-99a3-45f097f99cef.png">
